<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWidgetEditor</class>
 <widget class="QWidget" name="EmployeeWidgetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="4">
      <widget class="QLabel" name="l_salary">
       <property name="text">
        <string>Salary</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="l_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="l_function">
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="l_middlename">
       <property name="text">
        <string>Middlename</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="l_surname">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="pB_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QPushButton" name="pB_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5" colspan="2">
      <widget class="QSpinBox" name="sB_salary">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="5" colspan="2">
      <widget class="QLineEdit" name="lE_function"/>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QLineEdit" name="lE_middlename"/>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="QLineEdit" name="lE_surname"/>
     </item>
     <item row="0" column="5" colspan="2">
      <widget class="QLineEdit" name="lE_name"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
